server.go

```diff
@@ -2,79 +2,90 @@ package main
 
 import (
 	"fmt"
-	"github.com/BurntSushi/toml"
 	"html/template"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
+
+	"github.com/BurntSushi/toml"
```

Comment:
This is what gofmt and golint says you should do though. 
https://github.com/golang/go/wiki/CodeReviewComments#imports